Dockerfile.konflux

```diff
@@ -16,7 +16,7 @@ RUN cd ./cachi2/output/deps/generic && \
 
 ###############################################################################
 #latest tag
-FROM registry.redhat.io/ubi8/openjdk-17-runtime@sha256:44aa4ecdfa7a17513aff2fbd4368b87df1e3a9c576ec673ebaa11fa9afc3d79e as runtime
+FROM registry.access.redhat.com/ubi9/openjdk-17-runtime@sha256:046c2650c796bd5d98aa700315ab53c7bc2972ed70f6f7b96b843873e6d0c3bf as runtime
```

Comment:
This needs sync
https://github.com/red-hat-data-services/modelmesh/pull/310/files